experimental/dds/tree2/src/feature-libraries/flex-tree/lazyNode.ts

```diff
@@ -4,6 +4,7 @@
  */
 
 import { assert, unreachableCase } from "@fluidframework/core-utils";
+import { TypedEventEmitter } from "@fluid-internal/client-utils";
```

Comment:
I just realized tree has an `IEmitter` interface and probably an implementation; I'll explore moving to that intsead of bringing in `TypedEventEmitter`.

Reply:
Tried but it ends up causing a circular reference.

Reply:
Was that circular reference just called out by dep-cruiser? Or does it actually break runtime behavior? If only the latter, we can probably make a short-term exception. Consistent emitter patterns seems worth making the exception if we can.

Reply:
Did not run into circular references during a second attempt, so completed the move to `IEmitter`.